src/Sylius/Component/Mailer/Provider/EmailProvider.php

```diff
@@ -59,10 +59,6 @@ public function getEmail($code)
             $email = $this->getEmailFromConfiguration($code);
         }
 
-        if (null === $email) {
```

Comment:
Because at the method getEmailFromConfiguration is already doing that check. If it could not find the template code, it would throw an exception so i recon this checking is redundant. am i wrong?